core/src/test/scala/doric/Equalities.scala

```diff
@@ -1,42 +1,55 @@
 package doric
 
-import org.scalactic._
-import TripleEquals._
-import Tolerance._
 import java.sql.Date
 import java.time.{Instant, LocalDate}
+import org.scalactic._
+import org.scalactic.Tolerance._
+import org.scalactic.TripleEquals._
 
 object Equalities {
 
-  implicit def eqOptional[O: Equality]: Equality[Option[O]] = {
-    case (Some(x), Some(y)) => x === y
-    case (None, None)       => true
-    case _                  => false
-  }
-
-  implicit def eqList[O: Equality]: Equality[List[O]] = {
-    case (a: List[O], b: List[Any]) =>
-      a.zip(b).forall(x => x._1 === x._2)
-    case _ => false
+  implicit def eqOptional[O: Equality]: Equality[Option[O]] =
```

Comment:
why did you change this? is it not the same?

Reply:
Scala 2.11 doesn't have this way of declare a single function interface so we have to expand it :(

Reply:
💩